docs/node-operation/itn-manual.md

```diff
@@ -0,0 +1,210 @@
+a platform to deeply engage with Orcfax.
+
+## ITN rollout
+
+The ITN will progress through four phases, each adding additional layers of
+validator node functionality:
+
+### Phase 1: Data collection for CEXes
+
+During this phase, Validators will begin decentralizing the collection function
+of the Oracle network. Validators will initially run a limited suite of software
+responsible for the collection of data from centralized exchanges which have
+been identified as primary sources of information. Validators will then
+participate in funneling that data to the Orcfax centralized validator and
+publisher.
+
+### Phase 2: Incorporating DEX data
+
+Validators will now be able to collect the on-chain data from Cardano DEXes,
+which underpin Orcfax CNT feeds, and will do so by leveraging their own instance
+of a cardano node.
+
+Validators will continue to send all data to the Orcfax centralized validator
+and publisher.
+
+### Phase 3: Decentralized validator protocol
+
+During this phase, validators will begin decentralizing the validation function
+of the Orcfax network by running validator software. However, the publication of
+Fact Statements to the blockchain will remain controlled by the Orcfax
+centralized publisher.
+
+This phase will also see the implementation of a consensus mechanism which will
+enable the network to elect a single validator for submitting a verified Fact
+Statement to the Orcfax’s publisher.
+
+### Phase 4: Decentralized publication & Mainnet soft launch
+
+The final phase will see all validators with the functional ability to
+participate in collection, validation and publication on-chain (when nominated).
+
+During this phase, the consensus mechanism will be expanded so as to select a
+quorum of validators with approved data in order to then elect one to publish
+their Fact Statement on-chain and be rewarded for it.
+
+After this phase has been tested successfully, we will fully move to Mainnet.
+
+## Joining the Incentivized Testnet
+
+Commitment will key for those participating in the ITN. We expect participants
+to engage earnestly with the process, dedicating the necessary time and energy.
+This will include actively providing feedback that will be instrumental in
+refining and strengthening the Mainnet protocol.
+
+To join the ITN, participants must meet the following:
+
+### Validator License
+
+Initially, participants may only be required to hold a validator license on a
+Mainnet wallet. However, as we progress through the more advanced phases of the
+ITN, participants will also need one on Testnet.
+
+Holding a validator license on Mainnet will allow participants to mint one on
+Testnet.
+
+```text
+NB
+ITN rewards cannot be earned without a valid license.
+```
+
+### 500,000 FACT deposit
+
+Participants need to maintain this balance in their Mainnet wallet, which will
+later need to be secured in a smart contract on the Testnet with tFACT
+(Testnet FACT).
+
+### Hardware and Infrastructure
```

Comment:
This is a good approach (some info here and a separate technical requirements section). 

Given that though, I'd be tempted to keep this information less detailed, and say something like:

* min requirements (phase 1) are, single CPU, 2GB ram.
* recommended requirements (phase 2 onwards) are 32GB ram, 8vcpu, 600GB disk.

For more information see the technical manual.

Reply:
NB. we could also add a footnote that says something like: specs are based on the minimal viable digital ocean servers operated by Orcfax?